zerver/forms.py

```diff
@@ -349,3 +349,13 @@ def clean_emails(self) -> List[str]:
             raise forms.ValidationError(_("Please enter at most 10 emails."))
 
         return emails
+
+class GoToRealmForm(forms.Form):
```

Comment:
Maybe `RealmRedirect` instead of `GoToRealm`?
A bit of a pain to change, but hopefully not too much since the names are mostly concentrated in one commit.